reference/orders.v2.oas2.yml

```diff
@@ -4295,6 +4305,17 @@ components:
                 type: integer
               value:
                 type: string
+                description: |-
+                  Depending on the option type, value can be one of the following:
+                  - The variant option value id or the modifier value id for modifier types with a list of defined values, such as drop down or checkbox modifiers.
+                  - The modifier value for modifier types without a list of defined values, such as text field or date field modifiers.
+
+                  Notes:
+                  - The following option types are not currently supported via api:
+                    - File Upload
+                    - Pick List
+                  - For date modifiers use either the `YYYY-MM-DDThh:mm:ss±hh:mm` or the `YYYY-MM-DD` ISO-8601 formats. The date field modifier values are saved and retuned as timestamps. For values entered using the YYYY-MM-DD format the store timezone is used when creating the timestamp.
```

Comment:
For date modifiers use either the `YYYY-MM-DDThh:mm:ss±hh:mm` or the `YYYY-MM-DD` ISO-8601 formats. The date field modifier values are saved and returned as timestamps. For values entered using the YYYY-MM-DD format, use the store timezone when creating the timestamp.

Reply:
> For values entered using the YYYY-MM-DD format, use the store timezone when creating the timestamp.

We are doing the timestamp creation in the server. What I meant is that if they use the date only format we will use the store timezone to convert it to timestamp, but if they provide the timezone using the YYYY-MM-DDThh:mm:ss±hh:mm the provided timezone is respected.

Reply:
Thanks Dornah, this text should be fine